pyclam/manifold.py

```diff
@@ -105,6 +105,12 @@ def __contains__(self, point: Data) -> bool:
         """ Check weather the given point could be inside this cluster. """
         return self.overlaps(point=point, radius=0.)
 
+    def jaccard(self, other: 'Cluster') -> float:
+        """ The Jaccard Index between two Clusters. """
```

Comment:
I think a little text about what the jaccard index is would be nice to provide, but certainly not necessary.